clap_generate/README.md

```diff
@@ -1 +1,17 @@
 # clap_generate
+
+[![Crates.io](https://img.shields.io/crates/v/clap_generate?style=flat-square)](https://crates.io/crates/clap_generate)
+[![Crates.io](https://img.shields.io/crates/d/clap_generate?style=flat-square)](https://crates.io/crates/clap_generate)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue?style=flat-square)](https://github.com/clap-rs/clap/blob/master/LICENSE-APACHE)
+[![License](https://img.shields.io/badge/license-MIT-blue?style=flat-square)](https://github.com/clap-rs/clap/blob/master/LICENSE-MIT)
+[![Build Status](https://img.shields.io/github/workflow/status/clap-rs/clap/CI/master?style=flat-square)](https://github.com/clap-rs/clap/actions/workflows/ci.yml?query=branch%3Amaster)
+[![Coverage Status](https://img.shields.io/coveralls/github/clap-rs/clap/master?style=flat-square)](https://coveralls.io/github/clap-rs/clap?branch=master)
+[![Contributors](https://img.shields.io/github/contributors/clap-rs/clap?style=flat-square)](https://github.com/clap-rs/clap/graphs/contributors)
```

Comment:
We don't need these badges.

Reply:
I've removed the more generic ones but I think license ones still make sense (I see one was included, one was excluded in this range)